src/_gettsim/config.py

```diff
@@ -103,6 +103,11 @@ def set_array_backend(backend: str):
         "description": "A couple whose members are deemed to be responsible for each"
         " other.",
     },
+    "wth": {
```

Comment:
```suggestion
    "wthh": {
```
We'll hardly need it, better make it resemble the hh a little more.